### ISSUE TITLE
About Using Semantic Embedding Cue

### ISSUE THREAD

**Issue by coldmanck** [GriffinLiang/vrd-dsr#15]
Hi @GriffinLiang 

Thank you for you fantastic work again. I would like to utilize the fused feature vector to apply it to my dataset (in specifc, COCO). However it seems the model is dataset specific -- only apply to VRD which the number of classes is 100 when I want to retrieve the feature, the error mainly results from [semantic embedding part](https://github.com/GriffinLiang/vrd-dsr/blob/cd5db9611139a6c6fa2d2a5ec042a6388148102c/lib/nets/Vrd_Model.py#L127). As COCO does not have relationship labels, it's currently impossible to re-train the model on it. 

You indicated in the paper that you used off-the-shelf language model to acquire the word representations, so I am wondering how can I get the semantic embedding directly?

**Comment by GriffinLiang** [comment]
I use word2vec model. You can refer to the following link https://radimrehurek.com/gensim/models/word2vec.html.

**Comment by coldmanck** [comment]
Hi @GriffinLiang Thank you for your response. Could you please share your snippet of creating the features for reference purpose? :)

**Comment by GriffinLiang** [comment]
You should load a pretrained model, and get the representation of an input word. 
`model = Word2Vec.load(fname) `
`model.wv['computer']`

You can get the pretrained model following this [blog](http://mccormickml.com/2016/04/12/googles-pretrained-word2vec-model-in-python/).

**Comment by coldmanck** [comment]
Thanks for the explanation!
